front/src/views/Schedule.vue

```diff
@@ -98,31 +101,24 @@ export default {
             defaultGifs: [
                 {
                     day: 'Monday',
```

Comment:
maybe we can delete this tab and get days from Date ? wdyt ?

Reply:
Yes it can be a refactor ?

Reply:
for sure!